### PR TITLE
Add comments, clean up organization of make specs

### PR DESCRIPTION
**Pull request by collijk**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,9 +19,6 @@
     install_requirements = [
         'vivarium==0.9.3',
         'vivarium_public_health==0.10.4',
-        'gbd_mapping==2.1.0',
```
Comment (collijk) [inline setup.py:22]: Doing a complete separation from any inputs stuff for this project.

File: src/vivarium_gates_shigella_vaccine/paths.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from pathlib import Path
 
-from vivarium_gates_shigella_vaccine.globals import PROJECT_NAME
```
Comment (collijk) [inline src/vivarium_gates_shigella_vaccine/paths.py:3]: I prefer to import the entire globals module when we use it.  Help's distinguish project globals from any module defined globals.

File: src/vivarium_gates_shigella_vaccine/tools/cli.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+              help='Location to make specification for.')
+@click.option('-o', '--output-dir',
+              default=str(paths.MODEL_SPEC_DIR),
+              show_default=True,
+              type=click.Path(exists=True),
+              help='Specify an output directory. Directory must exist.')
+@click.option('-v', 'verbose',
+              count=True,
+              help='Configure logging verbosity.')
+def make_specs(template: str, location: str, output_dir: str, verbose: int) -> None:
```
Comment (collijk) [inline src/vivarium_gates_shigella_vaccine/tools/cli.py]: See MIC-1500

File: src/vivarium_gates_shigella_vaccine/tools/logging.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import sys
+from typing import TextIO
+
+from loguru import logger
+
+
+def add_logging_sink(sink: TextIO, verbose: int, colorize: bool = False, serialize: bool = False):
```
Comment (collijk) [inline src/vivarium_gates_shigella_vaccine/tools/logging.py]: See MIC-1499

File: src/vivarium_gates_shigella_vaccine/tools/make_specs.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+from jinja2 import Template
+from loguru import logger
+
+from vivarium_gates_shigella_vaccine import globals as project_globals
+from vivarium_gates_shigella_vaccine.utilites import sanitize_location
+
+
+def build_model_specifications(template: str, location: str, output_dir: str):
```
Comment (collijk) [inline src/vivarium_gates_shigella_vaccine/tools/make_specs.py:10]: This update should be part of MIC-1500

File: src/vivarium_gates_shigella_vaccine/tools/cli.py [hunk 6]
```diff
@@ -0,0 +1,68 @@
+"""
```
Comment (halfhorst) [inline src/vivarium_gates_shigella_vaccine/tools/cli.py:1]: I thought we wanted single cli.py at library src root

File: src/vivarium_gates_shigella_vaccine/tools/make_specs.py [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    """
+    template = Path(template)
+    output_dir = Path(output_dir)
+
+    if location == 'all':
+        locations = project_globals.LOCATIONS
+    elif location in project_globals.LOCATIONS:
+        locations = [location]
+    else:
+        raise ValueError(f'Location must be one of {project_globals.LOCATIONS}. '
```
Comment (halfhorst) [inline src/vivarium_gates_shigella_vaccine/tools/make_specs.py]: location can be 'all' also

**Review by halfhorst — APPROVED** [review]
One error message needs additional info

File: src/vivarium_gates_shigella_vaccine/tools/make_specs.py (hunk 7 above)
Comment (halfhorst) [reply src/vivarium_gates_shigella_vaccine/tools/make_specs.py]: @collijk this comment still seems relevant

**Pull request merged**